### PR TITLE
Run in foreground by default

### PR DESCRIPTION
**Pull request by m3nu**
This changes the default startup behavior from forking to the background to always start in foreground unless the existing argument `--daemonize` is used. Reasons for this change:

- Forking a Python process is broken on macOS. (see [1](https://bugs.python.org/issue33725) and [2](https://docs.python.org/3/library/multiprocessing.html#multiprocessing-start-methods)). There are rendering issues, random hangs or file access errors (seen with sqlite).
- Most uses seem to need it running in foreground: starting from Flatpak, starting from a macOS bundle, debugging.
- It removes some overly complicated startup conditions.
- There was always some confusion when users didn't see the system tray icon after starting it the first time.

Summary of changes:

- remove `--foreground` argument
- always run in foreground, unless existing `--daemonize` arg is used
- if no settings exist, open main window by default (can be changed under *Misc*)
- change description of `foreground` option to make it more clear. "Open main window on startup"
- change description of `--daemonize` arg to make it clear that the application will fork and never open a window (overrides `foreground` setting).

I hope this doesn't break anything. If there is anything else that can be simplified here, let me know.

### CODE REVIEW

**Comment by Hofer-Julian** [comment]
Travis reports flake8 errors which are probably easy to fix.

**Comment by m3nu** [comment]
Yeah. Fixed those.

**Comment by m3nu** [comment]
Will merge this soon, if all in favor.

**Comment by Hofer-Julian** [comment]
Will check today in the evening

-------- Original-Nachricht --------
An 28. Nov. 2019, 10:47, Manuel Riel schrieb:

File: src/vorta/models.py [hunk 1]
```diff
@@ -220,9 +220,9 @@ def get_misc_settings():
                                 'Automatically start Vorta at login')
         },
         {
-            'key': 'foreground', 'value': False, 'type': 'checkbox',
+            'key': 'foreground', 'value': True, 'type': 'checkbox',
             'label': trans_late('settings',
-                                'Run Vorta in the foreground when started manually')
+                                'Open main window on startup')
```
[the remark this answers is not in the document]
Comment (m3nu) [reply src/vorta/models.py]: The `--daemonize` option will still prevent opening the window, as before, overriding the user option under *Misc*.

https://github.com/borgbase/vorta/pull/380/files#diff-a25534ad739930813e5a6db7b45808d2R57

[the remark this answers is not in the document]
Comment (Hofer-Julian) [reply src/vorta/models.py]: I still think that "Run Vorta in the foreground" when started manually describes this option better than "open maon window on startup", but I am fine with your description if you prefer it.

**Pull request merged**